### PR TITLE
fix(trip-details): support routes without a short name

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
Co-authored-by: daniel-heppner-ibigroup <Daniel.heppner@ibigroup.com>

Some routes don't have a short name. This PR allows the fare table to handle these routes.

### CODE REVIEW

File: packages/itinerary-body/src/__mocks__/itineraries/otp2-with-fareproducts.json [hunk 1]
```diff
@@ -728,7 +728,6 @@
         "length": 368
       },
       "steps": [],
-      "routeShortName": "101",
```
Comment (AdrianaCeric) [inline packages/itinerary-body/src/__mocks__/itineraries/otp2-with-fareproducts.json:731]: Was this removed for testing?

**Review by AdrianaCeric — APPROVED** [review]
👍

File: packages/itinerary-body/src/__mocks__/itineraries/otp2-with-fareproducts.json (hunk 1 above)
Comment (miles-grant-ibigroup) [reply packages/itinerary-body/src/__mocks__/itineraries/otp2-with-fareproducts.json:731]: Yes! The story now shows how we handle a route with no short name

**Review by philip-cline — APPROVED** [review]
Have we considered the case where leg.route includes an insanely long route_id?

**Comment by miles-grant-ibigroup** [comment]
A very long route id and no long or short name sounds like a data issue to me

**Pull request merged**